ai_diffusion/workflow.py

```diff
@@ -150,7 +150,7 @@ def __init__(self, is_active=False, strength=0.3, seed=None):
 
 
 def _sampler_params(
-    style: Style, clip_vision=False, upscale=False, live=LiveParams()
+    style: Style, clip_vision=False, upscale=False, live=LiveParams(), strength: float = None
```

Comment:
* `strength: float = None` won't type-check - but it's also unclear why default is `None`, it can just be `1.0`?
* maybe rename `upscale=False` to `advanced=True` to indicate that it will create params for advanced sampler vs regular sampler. It doesn't have anything directly to do with upscale anymore.

Reply:
Done. You're right, its clearer this way.